.github/workflows/main.yml

```diff
@@ -60,18 +60,7 @@ jobs:
           # doesn't quite work. Perhaps our double/sparse checkout is to blame?
           git config --global --add safe.directory '*'
 
-          for moddir in /usr/lib/modules /lib/modules; do
-            if [ -e "$moddir" ]; then
-              kernel=$(find "$moddir" -maxdepth 1 -mindepth 1 -type d -exec basename {} \;)
-              break
-            fi
-          done
-
-          # There should be one entry - the kernel we installed
-          if [ $(echo "$kernel" | wc -l) -ne 1 ]; then
-            echo >&2 "Error: exactly one kernel must be installed"
-          fi
-          echo "KDIR=$moddir/$kernel/build" >> "$GITHUB_ENV"
+          scripts/print-kdir.sh >> "$GITHUB_ENV"
```

Comment:
like the ./.github/actions/setup-ubuntu ... I  think we should keep this in .github/ as I don't want `print-kdir.sh` being used outside of this scope... `./.github/print-kdir.sh` would be ok

Reply:
That is a simplification for a container environment that relies on having just one kernel installed.